docs/_posts/2017-01-18-jekyll-3-4-0-released.markdown

```diff
@@ -20,7 +20,7 @@ even bigger fan of [`group_by_exp`](/docs/filters/).
 And [lots and lots more!](/docs/history/#v3-4-0)
 
 This update was made possible by the dedicated efforts of our excellent
-contributors: Frank Taillandier, Tom Johnson, Ashwin Maroli, Parker Moore, Pat Hawks, alexmalik, Thiago Arrais, Nursen, Eldritch Cheese, XhmikosR, Roger Ogden, Ricardo N Feliciano, Nicolas Hoizey, Max Chadwick, Kenton Hansen, Josh Habdas, jona, Zlatan Vasović, yoostk, Tunghsiao Liu, Tim Banks, Skylar Challand, Rob Crocombe, Purplecarrot, muratayusuke, Longwelwind, Kurt Anderson, kimbaudi, Kevin Wojniak, Ivan Dmitrievsky, Hugo, Florian Thomas, Fabrice Laporte, Don Denton, Dmitrii Evdokimov, Dean Attali, Chayoung You, Chase, brainscript, BlueberryFoxtrot, Alexey Rogachev, Ajay Karwal.
+contributors: Frank Taillandier, Tom Johnson, Ashwin Maroli, Parker Moore, Pat Hawks, alexmalik, Thiago Arrais, Nursen, Eldritch Cheese, XhmikosR, Roger Ogden, Ricardo N Feliciano, Nicolas Hoizey, Max Chadwick, Kenton Hansen, Josh Habdas, penny, Zlatan Vasović, yoostk, Tunghsiao Liu, Tim Banks, Skylar Challand, Rob Crocombe, Purplecarrot, muratayusuke, Longwelwind, Kurt Anderson, kimbaudi, Kevin Wojniak, Ivan Dmitrievsky, Hugo, Florian Thomas, Fabrice Laporte, Don Denton, Dmitrii Evdokimov, Dean Attali, Chayoung You, Chase, brainscript, BlueberryFoxtrot, Alexey Rogachev, Ajay Karwal.
```

Comment:
@pnn Contributors are generated from git, you might want to update your config ?